### PR TITLE
Further Updates to the Docker Build Script

### PR DESCRIPTION
**Pull request by DukeDavis12**
Updates to Docker Build Script.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: utils/docker_build/build.sh [hunk 1]
```diff
@@ -6,10 +6,8 @@
 REMOTE_URL=https://github.com/secure-device-onboard/client-sdk.git
 REMOTE_BRANCH=master
 
-git clone http://github.com/intel/safestringlib.git
-cd safestringlib/ && make
-cd .. && rm -f CMakeCache.txt
-export SAFESTRING_ROOT=/home/sdouser/client-sdk/safestringlib
+rm -f CMakeCache.txt
+export SAFESTRING_ROOT=/home/sdouser/safestringlib
```
Comment (AdithyaBaglody) [inline utils/docker_build/build.sh:10]: is this still required?
because env is already set.

Comment (DukeDavis12) [reply utils/docker_build/build.sh:10]: No. Can be removed.

Comment (trbehera) [reply utils/docker_build/build.sh]: @DukeDavis12  Could you test and repost?

Comment (DukeDavis12) [reply utils/docker_build/build.sh]: Yes.. tested and reposted.

**Review by AdithyaBaglody — APPROVED** [review]

**Review by trbehera — APPROVED** [review]

**Pull request merged**